magda-web-client/src/Components/Dataset/Add/SpatialAreaInput/TerritorySelect.tsx

```diff
@@ -50,8 +50,8 @@ const TerritorySelect: FunctionComponent<PropsType> = props => {
     const { countryRegion } = props;
 
     const placeHolderText = countryRegion
-        ? "Please select country..."
```

Comment:
@sajidanower23 Good pick-up & Nice job~
The territory dropdown is actually the `Offshore Remote Territories` dropdown.
<img width="621" alt="image" src="https://user-images.githubusercontent.com/674387/66798648-6b6e5a00-ef5a-11e9-8f7c-7bdffee832d3.png">
Probably should be `Please select a offshore Remote Territory`?
@alyce Could you please help to give some feedback?

Reply:
Yes could it say "Please select an Offshore Remote Territory"

Reply:
Thanks @alyce . Done :+1: